api/prototool.yaml

```diff
@@ -11,6 +11,13 @@ protoc:
     - ../vendor
 
 lint:
+  ignores:
+    - id: ENUM_ZERO_VALUES_INVALID
+      files:
+        - managementpb/metrics.proto
+    - id: ENUM_ZERO_VALUES_INVALID_EXCEPT_MESSAGE
+      files:
+        - managementpb/metrics.proto
```

Comment:
That's very nice that we can do it only for that file

Reply:
also, i've tried to use struct comments from documentation, but it didn't work for some reason.